_posts/2023-12-13-release-0_4_0.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "kroxylicious release 0.4.0"
+date:   2023-12-14 00:00:00 +1300
+author: "Keith Wall"
+author_url: "https://www.github.com/k-wall"
+categories:  releases kroxylcious topic-encryption
+---
+
+The Kroxylicious project is very pleased to announce the [release](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.4.0) of Kroxylicious 0.4.0.
+
+The big news is this release includes our `EnvelopeEncryption` filter which provides an **Encryption-at-Rest** solution for the Apache Kafka(tm) which is _transparent_ to both clients and brokers. We have more work planned to iterate on the implentation to reach production quality. Checkout our [0.5.0 milestone](https://github.com/kroxylicious/kroxylicious/milestones/0.5.0) for details.
+
+The `EnvelopeEncryption` filter works by intercepting all produce requests from applications and applies encryption to the kafka records as they pass through Kroxylicious. On the consume path, the reverse happens - the filter intercepts the fetch responses and decrypts the records before they are sent to the application.
+
+The Kafka Cluster *never* sees the plain text of your records.   You use filter configuration to specify which topic(s) you wish to be encrypted by which key.
```

Comment:
```suggestion
The Kafka Cluster *never* sees the plain text of your records. Filter configuration is used to specify which topic(s) should be encrypted by which key.
```

credit to @gracegrimwood for figuring out the right form of words.

Reply:
I have a few more things I would alter here, like using a single space between sentences
```suggestion
The Kafka Cluster *never* sees the plain text of encrypted records. Filter configuration is used to specify which topic(s) are to be encrypted by which key.
```